### PR TITLE
Movie carussel

### PR DESCRIPTION
**Pull request by iameru**
this includes #22 and makes the Pull Request #24 obsolete as this contains a better and improved version of the moviecard.

this is a caroussel with scroll snapping and the like. It was tough to implement and test in Storybook as we had to artificially make a limiting factor outside. will adjust if neessary when we implement it in #20

### CODE REVIEW

File: src/components/MovieCarussel.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+type props = {
+  movies: MovieProps[];
+};
+
+export function MovieCarussel({ movies }: props) {
+  return (
+    <div className="flex flex-col gap-5 w-full">
+      <h2 className="text-white font-bold">Upcoming Movies</h2>
+      <section className="overflow-x-scroll overscroll-contain snap-x flex space-x-10">
```
Comment (ReallyBoard123) [inline src/components/MovieCarussel.tsx:16]: see if you can find a way to hide the scroll bar

File: src/stories/MovieCard.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import react from "react";
+import { Meta, StoryObj } from "@storybook/react";
+import "../index.css";
+import { MovieCard } from "../components/MovieCard";
+
+const imageUrls = [
```
Comment (NoHara42) [inline src/stories/MovieCard.stories.tsx:6]: why are these hard-coded?
Please take these from the TMDB API, together with @Sebi900 
-> create a custom hook to fetch the movie covers from the API

**Review by NoHara42 — CHANGES_REQUESTED** [review]
See comments and chat with @Sebi900

File: src/stories/MovieCard.stories.tsx (hunk 2 above)
Comment (iameru) [reply src/stories/MovieCard.stories.tsx:6]: They are hardcoded as we dont use the API yet and they are just Testing cases for the storybook. When we work on the API access and implementation later I can adjust.

**Comment by iameru** [comment]
> @iameru please do not lump PRs together. Keep one PR per issue. It is not fun to review multiple issues in one PR.

I will, couldnt find an issue to work on though without dependencies on unreviewed ones. as they were very close attached (imagecard and imagecard carussel) I merged them.

File: src/stories/MovieCard.stories.tsx (hunk 2 above)
Comment (iameru) [reply src/stories/MovieCard.stories.tsx:6]: as we dont use API yet its not possible to change this. I"d suggest to continue with the hardcoded test cases for storybook and if the issue for the actual use of API is on the way ( f.e. as part of #20 ) I can also adjust the storybook.

File: src/components/MovieCarussel.tsx (hunk 1 above)
Comment (iameru) [reply src/components/MovieCarussel.tsx]: fixed in b08e5b0

File: src/stories/MovieCard.stories.tsx (hunk 2 above)
Comment (NoHara42) [reply src/stories/MovieCard.stories.tsx:6]: I'll create an issue for creating custom hooks that relate to the TMDB API

**Review by NoHara42 — CHANGES_REQUESTED** [review]
Blocked until the API custom hooks have been implemented, see related issue

**Review by NoHara42 — APPROVED** [review]
LGTM, continue mounting this to the Home and/or the Movies pages

**Pull request merged**